<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:rgb(0, 0, 0);
}
* {
    background-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="aboutpushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255, 255, 0);
	font-size: 35px;
	font-family: &quot;Arcade Classic&quot;;
    border: none;
}</string>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QPushButton" name="playpushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255, 255, 0);
	font-size: 35px;
	font-family: &quot;Arcade Classic&quot;;
    border: none;
}</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="replayPushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255, 255, 0);
	font-size: 35px;
	font-family: &quot;Arcade Classic&quot;;
    border: none;
}</string>
    </property>
    <property name="text">
     <string>Replay</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitpushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    color: rgb(255, 255, 0);
	font-size: 35px;
	font-family: &quot;Arcade Classic&quot;;
    border: none;
}</string>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>641</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: rgb(255, 255, 0);
	font-size: 120px;
	font-family: &quot;Arcade Classic&quot;;
    border: none;
    text-align: center;
}</string>
    </property>
    <property name="text">
     <string>PACMAN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
